File: Python/Matemática/1761 - Decoração Natalina.py
"""
Autor: Pierre Vieira
Data da submissão: 11/02/2020 23:08:50
"""
from math import tan
while True:
    try:
        a, b, c = map(float, input().split())
    except EOFError:
        break
    else:
        t = 5 * ((tan(a * (3.141592654/180)) * b) + c)
        print('{:.2f}'.format(t))

File: Kotlin/Matemática/1761 - Decoração Natalina.kt
/*
* Autor: Pierre Vieira
* Data de submissão: 18/03/2023 15:27:42
*/

fun main(args: Array<String>) {
    val pi = 3.141592654
    while (true) {
        val (a, b, c) = readLine()?.split(" ")?.map { it.toDouble() } ?: break
        val result: Double = 5 * ((tan(a * (pi / 180)) * b) + c)
        val resultString = String.format("%.2f", result)
        println(resultString)
    }
}